.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+<!--
+Thank you for your pull request. Please provide a description above and review
+the checklist below.
+
+Contributors guide: ./CONTRIBUTING.md
+-->
+
+## Checklist
+<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->
+
+- [ ] **DO** keep pull requests small so they can be easily reviewed.
+- [ ] **DO** make sure linting passes.
```

Comment:
Isn't this enforced by GitHub anyway?

Reply:
Also, I'd remove the `**DO**` since it's a to-do list anyway...

Reply:
Yes it is enforced. But this will be shown before a PR gets submitted. This might the contributor to halt and check again. Might safe a CI failure round trip.

Reply:
As for the `**DO**` and `**AVOID**`, let us see what others think.